api.go

```diff
@@ -575,8 +608,17 @@ func (c Client) newRequest(method string, metadata requestMetadata) (req *http.R
 		return nil, err
 	}
 
+	// Go net/http notoriously closes the request body.
+	// - The request Body, if non-nil, will be closed by the underlying Transport, even on errors.
```

Comment:
If golang's Transport module closes contentBody, why do we close it in executeMethod() ?